build_run.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+mvn package && clear && java -jar target/tictactoe-1.0-SNAPSHOT.jar
```

Comment:
This is nitpicking a bit, but you should make sure that each of your files ends in a newline: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline